graphql-dgs-subscriptions-sse/src/main/kotlin/com/netflix/graphql/dgs/subscriptions/sse/DgsSSESubscriptionHandler.kt

```diff
@@ -42,11 +42,11 @@ import java.util.*
 @RestController
 open class DgsSSESubscriptionHandler(open val dgsQueryExecutor: DgsQueryExecutor) {
     open val logger = LoggerFactory.getLogger(DgsSSESubscriptionHandler::class.java)
+    private val mapper = jacksonObjectMapper()
```

Comment:
why not inject these?

Reply:
Yeah Ii think that is the way to go ultimatley, but it requires a bit more care. For instance, I imagine we'd want qualify the mapper used internally in DGS, and not just use the default ObjectMapper, and does it make sense to expose a builder / customizer as a hook point for people to fiddle with the configuration, etc.

Reply:
Yes, we shouldn't inject the default one. Folks set those up for various reasons, not necessarily expecting the framework to use that as well. We've seen issues with this previously.

A DGS specific one would work.